podvm/Dockerfile.podvm

```diff
@@ -4,17 +4,22 @@
 #
 # Builds pod vm image inside container
 #
-ARG BUILDER_IMG="quay.io/confidential-containers/cloud-api-adaptor-podvm-builder"
+ARG BUILDER_IMG="quay.io/confidential-containers/podvm-builder-ubuntu"
 
 FROM ${BUILDER_IMG} AS podvm_builder
 
 ARG CLOUD_PROVIDER=libvirt
 ARG PODVM_DISTRO=ubuntu
 
-# If these are not provided as build args then the
-# default values assigned in the Makefile will get used
```

Comment:
should these be removed from the makefile?

Reply:
For running non-container builds these might be needed.